// statue-ssg/src/lib/index.ts
// place files you want to import through the `$lib` alias in this folder.

// Export all components for easy importing
export { default as Hero } from './components/Hero.svelte';
export { default as Stats } from './components/Stats.svelte';
export { default as Categories } from './components/Categories.svelte';
export { default as LatestContent } from './components/LatestContent.svelte';
export { default as PageHero } from './components/PageHero.svelte';
export { default as Mission } from './components/Mission.svelte';
export { default as Team } from './components/Team.svelte';
export { default as WhyChooseUs } from './components/WhyChooseUs.svelte';
export { default as CTA } from './components/CTA.svelte';
export { default as DirectoryHeader } from './components/DirectoryHeader.svelte';
export { default as SubDirectories } from './components/SubDirectories.svelte';
export { default as DirectoryContent } from './components/DirectoryContent.svelte';
export { default as ContentHeader } from './components/ContentHeader.svelte';
export { default as ContentBody } from './components/ContentBody.svelte';
export { default as NavigationBar } from './components/NavigationBar.svelte';
export { default as Footer } from './components/Footer.svelte';
export { default as Sitemap } from './components/Sitemap.svelte';
export { default as Warning } from './components/Warning.svelte';
export { default as Signup } from './components/Signup.svelte';
export { default as BuiltBy } from './components/BuiltBy.svelte';
export { default as Search } from './components/Search.svelte';
export { default as PdfViewer } from './components/PdfViewer.svelte';

export { default as CollapsibleTree } from './components/CollapsibleTree.svelte';

// Documentation components
export { default as DocsSidebar } from './components/DocsSidebar.svelte';
export { default as TableOfContents } from './components/TableOfContents.svelte';
export { default as DocsLayout } from './components/DocsLayout.svelte';
export { default as DocsContent } from './components/DocsContent.svelte';
export { default as DocsDirectoryList } from './components/DocsDirectoryList.svelte';

// Blog components
export { default as AuthorAvatar } from './components/AuthorAvatar.svelte';
export { default as BlogCard } from './components/BlogCard.svelte';
export { default as BlogGrid } from './components/BlogGrid.svelte';
export { default as BlogHeader } from './components/BlogHeader.svelte';
export { default as BlogLayout } from './components/BlogLayout.svelte';
export { default as BlogPostContent } from './components/BlogPostContent.svelte';
export { default as BlogPostHeader } from './components/BlogPostHeader.svelte';
export { default as BlogPostLayout } from './components/BlogPostLayout.svelte';

export { default as AccordionMenu } from './components/AccordionMenu.svelte';
export { default as CopyCommand } from './components/CopyCommand.svelte';
export { default as GitHubStats } from './components/GitHubStats.svelte';
export { default as ImageCarousel } from './components/ImageCarousel.svelte';
export { default as NotificationBanner } from './components/NotificationBanner.svelte';
export { default as PricingSection } from './components/PricingSection.svelte';
export { default as ScrollingLogos } from './components/ScrollingLogos.svelte';
export { default as SlidingComparison } from './components/SlidingComparison.svelte';
export { default as SlidingTestimonial } from './components/SlidingTestimonial.svelte';
